spec/index.html

```diff
@@ -153,1920 +190,1795 @@ <h2>Semantic Extensions and Entailment Regimes</h2>
+          it is consistent to assert that something is both an <code>rdf:Bag</code> and an <code>rdf:Seq</code>.
+          There is no assumption that containers are gap-free, so that for example
+        </p>
+        <pre class="nohighlight">_:xxx rdf:type rdf:Seq.
+          _:xxx rdf:_1 ex:a .
+          _:xxx rdf:_3 ex:c .
+        </pre>
+        <p>does not entail</p>
+        <p><code>_:xxx rdf:_2 _:yyy .</code></p>
+        <p>There is no way in RDF to assert
+          that a container contains only a fixed number of members. This is a
+          reflection of the fact that it is always consistent to add a triple
+          to a graph asserting a membership property of any container. And
+          finally, there is no built-in assumption that an RDF container has
+          only finitely many members.
+        </p>
+      </section>
+      <section id="collections">
+        <span id="rdf-collections">
+          <!-- Alternative identifier -->
+        </span>
+        <h4>RDF collections</h4>
+        <section class="notoc">
+          <h6 class="defcap">RDF Collection Vocabulary</h6>
+          <div class="othertable">
+            <p><code>rdf:List rdf:first rdf:rest rdf:nil</code></p>
+          </div>
+        </section>
+        <p>RDF provides a vocabulary for describing collections, i.e.'list
+          structures', in terms of head-tail links. Collections differ from
+          containers in allowing branching structure and in having an
+          explicit terminator, allowing applications to determine the exact
+          set of items in the collection.
+        </p>
+        <p>As with containers, no special semantic conditions are imposed on this vocabulary
+          other than the type of <code>rdf:nil</code> being <code>rdf:List</code>. It
+          is intended for use typically in a context where a container is described using
+          blank nodes to connect a 'well-formed' sequence of items, each described by
+          two triples of the form
+          <code><br>
+          <br>
+          _:c1 rdf:first aaa .<br>
+          _:c1 rdf:rest _:c2 .</code>
+        </p>
+        <p>where the final item is indicated by the use of <code>rdf:nil</code> as the
+          value of the property <code>rdf:rest</code>. In a familiar convention, <code>rdf:nil</code>
+          can be thought of as the empty collection. Any such graph amounts to an assertion
+          that the collection exists, and since the members of the collection can be determined
+          by inspection, this is often sufficient to enable applications to determine
+          what is meant. The semantics does not require any collections
+          to exist other than those mentioned explicitly in a graph (and the empty collection).
+          For example, the existence of a collection containing two items does not automatically
+          guarantee that the similar collection with the items permuted also exists:
+          <br>
```

Comment:
The doc currently includes the `pre` block within a `p`, which is not allowed.

```suggestion
        </p>
```